PhysicsTools/PatAlgos/python/slimming/packedPFCandidates_cfi.py

```diff
@@ -12,5 +12,7 @@
       cms.InputTag("inclusiveCandidateSecondaryVertices"),
       cms.InputTag("inclusiveCandidateSecondaryVerticesCvsL"),
       ),      
-    minPtForTrackProperties = cms.double(0.95)
+    minPtForTrackProperties = cms.double(0.95),
+    covarianceVersion = cms.int32(1), #so far: 0 is Phase0, 1 is Phase1   
+    covariancePackingSchemas = cms.vint32(1,257,513,769,0)   
```

Comment:
some documentation of these magic numbers should be added, probably in a readme file in DataFormats/PatCandidates or in DataFormats/PatCandidates/doc/PatCandidates.doc